src/index.html

```diff
@@ -7,8 +7,180 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
+    <header class="blue-box">
+
+      <div class="header">
+        <img class="header__logo" src="./images/logo/dia-logo.svg" alt="Logo">
+        <nav class="header__nav-wrap">
+          <ul class="header__nav nav">
+            <li class="nav__item">
+              <a href="#" class="nav__link">Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">About us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Services</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Work</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Contact Us</a>
+            </li>
+          </ul>
+          <a href="#" class="nav__link nav__link--is-active">Hire us</a>
+        </nav>
+      </div>
+
+      <div class="intro-slider-wrap">
+        <div class="intro">
+            <h1 class="intro__heading">Global digital agency</h1>
+            <p class="intro__description">By the same illusion which lifts the horizon
+              of the sea to the level of the spectator on a hillside, the sable cloud beneath.
+            </p>
+            <button class="intro__button"><a href="#" class="intro__link">Learn more</a></button>
+        </div>
+        <div class="slider">
+          <div class="slider__menu">
+            <div class="slider__arrows-box">
+              <button class="slider__arrow-box"><img alt="arrow left" class="slider__arrow" src="./images/slider/arrow-left.svg"></button>
```

Comment:
не пиши все в один рядок
```suggestion
              <button class="slider__arrow-box">
                <img alt="arrow left" class="slider__arrow" src="./images/slider/arrow-left.svg">
              </button>
```